example-notebooks/air-quality-covid.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
<p>This cell could use some sort of description.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I've added some clarifying text before this to give context. I haven't, however, said anything about the unit here (mainly because the catalog itself doesn't say anything about it - I'd imagine it is parts per million or similar. Not sure if that level of detail is required, but might help. </p>